lib/flipper/api/v1/actions/boolean_gate.rb

```diff
@@ -0,0 +1,21 @@
+require 'flipper/api/action'
+
+module Flipper
+  module Api
+    module V1
+      module Actions
+        class BooleanGate < Api::Action
+          route %r{api/v1/features/[^/]*/(enable|disable)/?\Z}
+          
+          def put
+            feature_name = Rack::Utils.unescape(request.path.split("/")[-2])
+            feature = flipper[feature_name.to_sym]
+            action = Rack::Utils.unescape(request.path.split("/").last)
```

Comment:
route_parts = request.path.split("/") and then we can re-use that for feature_name and action.